### PR TITLE
Franz Fernando

### PR DESCRIPTION
**Pull request by FJF616**
initial commit completes written portion

### CODE REVIEW

**Review by BlackBelt2817 — COMMENTED** [review]
Nice work, Franz. Your 1d looks good and your questions are thorough. Keep it up!

File: 1d-life/src/1d-life.js [hunk 1]
```diff
@@ -41,7 +41,18 @@
...
-    return 0; // instead of this
+   
+    let left = 0; let origin = lifeState[x]; let right= 0;
+        if ( x-1 > 0) {
+          lifeState[x-1] === 1 ? left = 1 : left = 0;
+        }
+        if ( x+1 < lifeState.length) {
+          lifeState[x+1] === 1 ? right = 1 : right = 0;
+        }
+        if (left & right & origin) {
```
Comment (TheDeterminator) [inline 1d-life/src/1d-life.js:52]: Nice, I like the bitwise operations here and on line 55.

File: security/PERSONAL SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿# **PERSONAL SECURITY**
+
+The step I took for improving my personal security was install i2p on my computer similar to the TOR network, the I2P network provides strong privacy protections for communication over the Internet. Many activities that would risk your privacy on the public Internet can be conducted anonymously inside I2P. The table below will summarize some of the benefits of using I2P.
+
+
+| Email                                    | Web browsing                             | Blogging and forums         | Website hosting                 | Real-time chat                    | Decentralized File Storage               |
```
Comment (TheDeterminator) [inline security/PERSONAL SECURITY.md:6]: Nice! I really like how you use a table here.

File: security/THREATMODEL.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿# **THREATMODEL**  
+*the purpose of this article is to address a posible security threat that has the postential to compromise secure, private, or personal information for all personnel that must perform work duties abroad*
```
Comment (TheDeterminator) [inline security/THREATMODEL.md:2]: sp *possible

Comment (TheDeterminator) [inline security/THREATMODEL.md:2]: Also good job establishing what you will talk about in this article.

File: security/THREATMODEL.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+As summarized in the table above, there is ample reason for concern, for our collegues that work abroad from time to time.  When we are out on our travels, we will many times either have lunch with clients or grab food at a food court while on the go.  This opens us up to attack, particularly while waiting in line to order or just standing looking at a menu.  An attacker needs but a few moments to execute an attack with his mobile device to get any information that is on the credit cards in our wallets.  He then sends this data to his partners who can be anywhere- and are sometimes in another state or country.  From there, it takes just a few moments more before they have produced a replica credit card to with our info, to be used to make fraudulent purchases.  The best and easiest way to prevent such an attack is to keep all credit cards in an anti static sleeve in order to block any form of communication with nearby devices.  We can also take measures to report our movements ahead of time to our credit holders so that they will be aware that certain purchases will only be made within a certain area and that any purchases made at the same time anywhere else are suspect.
```
Comment (TheDeterminator) [inline security/THREATMODEL.md:10]: Good summary

**Review by TheDeterminator — COMMENTED** [review]
Fernando, I really enjoyed reviewing your project and reading through your answers. Everything here is really solid. Great job!

**Pull request closed without merging**